install_files/securedrop-app-code/debian/copyright

```diff
@@ -1,9 +1,9 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: securedrop-proxy
```

Comment:
Great catch! We should probably change `Upstream-Name: securedrop-proxy` to `Upstream-Name: securedrop` as well.